plugins/obs-ffmpeg/obs-ffmpeg-nvenc.c

```diff
@@ -139,6 +139,12 @@ static bool nvenc_update(void *data, obs_data_t *settings)
 	int gpu = (int)obs_data_get_int(settings, "gpu");
 	bool cbr_override = obs_data_get_bool(settings, "cbr");
 	int bf = (int)obs_data_get_int(settings, "bf");
+	int spstrength = (int)obs_data_get_int(settings, "spatialaqstrength");
```

Comment:
If you want to abbreviate "Spatial AQ Strength" for the variable here, perhaps "saqstrength" would be slightly more appropriate?